src/warden/Cargo.toml

```diff
@@ -51,6 +51,12 @@ version = "0.2"
 features = ["auto-capture"]
 optional = true
 
+[target.'cfg(target_os = "macos")'.dependencies.gfx-backend-metal]
+path = "../../src/backend/metal"
+version = "0.2"
+features = ["signpost"]
```

Comment:
I don't think Warden needs signposts

Reply:
No, I was just blindly copying the "auto-capture" example, back when I couldn't work out how to pass feature flags to cargo.